common/common.py

```diff
@@ -0,0 +1,62 @@
+# Copyright © 2025 IBM
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
At some point we might want copyright/license checks.
Some files have verbatim, some none 
We can probably ensure we have the full license text within the module (not just the project?) and use SPDX in source?
Can be addressed in future though

Reply:
Yes, love it. For future. Feel free to create an issue for that. Thx